source/common/grpc/common.h

```diff
@@ -25,6 +25,34 @@ class Exception : public EnvoyException {
 
 class Common {
 public:
+  enum GrpcStatus {
```

Comment:
+1 to what @htuch said. I would like to re-use those for fault injection as well. Do we also have HTTP2 specific codes in a similar place (in addition to HTTP1)? All I see are HTTP1 codes in `include/envoy/http/codes.h`